src/about-this-guide.md

```diff
@@ -16,4 +16,11 @@ be found at the [GitHub repository]. If you find any mistakes in the
 guide, please file an issue about it, or even better, open a PR
 with a correction!
 
+## Other places to find information
+
+You are going to find [Rustc API docs site] useful and there's more
```

Comment:
Maybe this:

You might also find the following sites useful:

- [Rustc API docs] -- rustdoc documentation for the compiler
- [Forge] -- contains documentation about rust infrastructure, team procedures, and more
- [compiler-team] -- the home-base for the rust compiler team, with description of the team procedures, active working groups, and the team calendar.

Reply:
:+1: